src/Groups/GAPGroups.jl

```diff
@@ -756,123 +758,130 @@ function Base.rand(rng::Random.AbstractRNG, C::GroupConjClass{S,T}) where S wher
 end
 
 """
-    conjugacy_classes_subgroups(G::Group)
+    subgroup_classes(G::GAPGroup; order::T = ZZRingElem(-1)) where T <: IntegerUnion
 
-Return the vector of all conjugacy classes of subgroups of G.
+Return the vector of all conjugacy classes of subgroups of G or,
```

Comment:
```suggestion
Return the vector of all conjugacy classes of subgroups of `G` or,
```
Should it really be "the vector", not "a vector"? (Of course that's beyond the scope of this PR, but while we are at it...